core/src/findTransactionSignature.ts

```diff
@@ -1,9 +1,20 @@
 import { ConfirmedSignatureInfo, Connection, Finality, PublicKey, SignaturesForAddressOptions } from '@solana/web3.js';
 
+/** @ignore */
 export class FindTransactionSignatureError extends Error {
     name = 'FindTransactionSignatureError';
 }
 
+/**
+ * Find a transaction signature
+ *
+ * @throws if signature can't be found
+ *
+ * @param connection - A connection to the cluster.
+ * @param reference - A `PublicKey` that was included as a reference in the transaction.
+ * @param {SignaturesForAddressOptions} options - Options for getSignaturesForAddress.
+ * @param finality - A subset of Commitment levels, which are at least optimistically confirmed
```

Comment:
Nit: A user needs to know what are the possible levels. Should link Finality from web3